elastic/kibana
# [Security Solutions] Align global menu with security solutions menu

issue: https://github.com/elastic/kibana/issues/163458

## Summary

Update the global menu to reflect the latest changes to the security solution menu.


## How to test it?
* start ESS kibana
* Open security solution
* Compare the global menu and the security solutions menu
* It should display the same item order


<img width="234" alt="Screenshot 2023-08-21 at 10 48 56" src="https://github.com/elastic/kibana/assets/1490444/84716392-e15c-4377-9c55-51d8109afd40">
<img width="242" alt="Screenshot 2023-08-21 at 10 49 05" src="https://github.com/elastic/kibana/assets/1490444/89f7516e-48f1-4f61-8272-2f6dd662b53e">
